gotham/src/test/mod.rs

```diff
@@ -173,13 +171,19 @@ where
             .map(|_| ())
             .map_err(|_| ());
 
-        handle.spawn(f);
+        {
+            self.data.runtime.read().unwrap().spawn(f);
+        }
 
-        let client = Client::configure()
-            .connector(TestConnect {
-                stream: cell::RefCell::new(Some(cs)),
+        let client = Core::new()
```

Comment:
This looks like the crucial remaining usage of tokio_core, right?

Reply:
@nyarly yep! This is the one that has to stick around until Hyper 0.12 is adopted.